08-communicating-results.Rmd

```diff
@@ -35,36 +37,34 @@ This sentence provides a few key pieces of information for the audience:
 
-Including data tables and graphs are used to portray a large amount of data in a concise manner. Although discussing key findings in the text is important, it is easier for the audience to digest large amounts of data in graphical or table format. When used correctly, combining text, tables, and graphs is extremely powerful in presenting results. This section provides examples of using the {gtsummary} and {ggplot} packages to enhance the dissemination of results.
+Including data tables and graphs are used to portray a large amount of data in a concise manner. Although discussing key findings in the text is important, it is often easier for the audience to digest large amounts of data in graphical or table format. When used correctly, combining text, tables, and graphs is extremely powerful in presenting results. This section provides examples of using the {gtsummary} and {ggplot} packages to enhance the dissemination of results.
 
 ### Tables
 
-Tables are a great way to provide a large amount of data when we want the individual data points to be read. However, it is important to present tables in a readable format. Numbers should be aligned, and rows and columns should be easy to follow. Using key visualization techniques, we can create tables that are informative and nice to look at. Many packages can be used to create easy-to-read tables (e.g., {kable} + {kableExtra}, {gt}, {gtsummary}, {DT}, {formattable}, {flextable}, {reactable}). We will focus on {gt} and {gtsummary} here, but we encourage learning about others as they may have additional helpful components.
+Tables are a great way to provide a large amount of data when we want the individual data points to be read. However, it is important to present tables in a readable format. Numbers should be aligned, and rows and columns should be easy to follow. Using key visualization techniques, we can create tables that are informative and nice to look at. Many packages can be used to create easy-to-read tables (e.g., {kable} + {kableExtra}, {gt}, {gtsummary}, {DT}, {formattable}, {flextable}, {reactable}). We will focus on {gt} and {gtsummary} here, but we encourage learning about others as they may have additional helpful components. 
 
 Let's start by using some of the data we calculated earlier in this book. In Chapter \@ref(c05-descriptive-analysis), we looked at the total electric bill by Census region using the `recs` data:
 ```{r}
 #| label: results-table-raw
 bill_region <- recs_des %>%
   group_by(Region) %>%
-  cascade(elec_bill = survey_total(!is.na(DOLLAREL)))
+  cascade(elec_bill = survey_total(!is.na(DOLLAREL)))  
 
 bill_region
 ```
 
 The native output that R produces may work for initial viewing inside RStudio or when creating basic output with an R Markdown or Quarto document. However, when viewing these results in other publications, such as the print version of this book or for more official dissemination, adjusting the display can make it easier for users to follow. 
 
```

Comment:
We should explain why gt (uses tidyverse syntax, flexibility in construction, and the auxiliary packages like gtsummary that work well for survey analysts)